src/Commands/config/ConfigCommands.php

```diff
@@ -175,6 +175,85 @@ public function delete($config_name, $options = []) {
   }
 
   /**
+   * Display status of configuration (differences between stored and active configuration).
+   *
+   * @command config-status
+   * @param $label A config directory label (i.e. a key in \$config_directories array in settings.php).
+   * @interact-config-label
+   */
+  public function status($label = NULL) {
+    $directory = $this->getDirectory($label, '');
+    $storage = $this->getStorage($directory);
+    $change_list = $this->getChanges($storage);
+    if (empty($change_list)) {
+      $this->logger()->notice(dt('The active configuration is identical to the configuration in the export directory (!target).', array('!target' => $directory)));
+      return;
+    }
+    drush_print("Differences of the active config to the export directory:\n");
+    ConfigCommands::configChangesTablePrint($change_list);
+  }
+
+  /**
+   * Determine which configuration directory to use and return directory path.
+   *
+   * Directory path is determined based on the following precedence:
+   *   1. User-provided $directory.
+   *   2. Directory path corresponding to $label (mapped via $config_directories in settings.php).
+   *   3. Default sync directory
+   *
+   * @param string $label
+   *   A configuration directory label.
+   * @param string $directory
+   *   A configuration directory.
+   */
+  function getDirectory($label, $directory) {
+    // If the user provided a directory, use it.
+    if (!empty($directory)) {
+      if ($directory === TRUE) {
```

Comment:
I don't understand how `$directory` could ever be a boolean, but I'm preserving this legacy code path just in case I've missed something.